content/reference-data-api.md

```diff
@@ -1752,6 +1757,207 @@ Response:
 
 
 
+## Get Top Currencies ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/network/topCurrencies.js "Source")
+
+Returns the top currencies on the Ripple Consensus Ledger over a rolling 30 day window. The currencies are ordered from highest ranked to lowest, where the rank is determined by the volume and count of transactions and the number of unique counterparties. By default, returns the top currencies for the most recent date available. You can query historical data by date. _(New in [v2.1.0][])_
+
+
+#### Request Format ####
+
+<!--<div class='multicode'>-->
+
+*Most Recent*
+
+```
+GET /v2/network/top_currencies
+```
+
+*By Date*
+
+```
+GET /v2/network/top_currencies/2016-01-01
```

Comment:
Maybe, per the [API Template](https://ripplelabs.atlassian.net/wiki/display/DOCS/REST+API+Method+Template), add a note about this method not taking any URL parameters.